cloudshell_utilities/whitelist_for_wfh.sh

```diff
@@ -0,0 +1,27 @@
+if [[ -z "$1" ]]; then
+  echo "Missing required argument: WFH IP Address"
+  exit 1
+fi
+if [[ -z "$2" ]]; then
+  echo "Missing required argument: WFH Person's Name"
+  exit 1
+fi
+WFH_IP=$1
+WFH_NAME=$2
+pushd "${0%/*}" || exit 1
+
+echo "Whitelisting IP: ${WFH_IP} with name ${WFH_NAME} WFH"
+gcloud config set project census-rm-whitelodge
+gcloud container clusters get-credentials rm-k8s-cluster --region europe-west2 --project census-rm-whitelodge
+pipenv run python whitelist_service_ip.py $WFH_IP case-api || exit 1
```

Comment:
not sure this will work, the default IP's are static internal ones, so even if we add a whitelist they're still not visible, i think this is we a created a second k8s case-api service for this 🤔

Reply:
have removed it until we create a 2nd case-api service

Reply:
i've created a [PR](https://github.com/ONSdigital/census-rm-kubernetes/pull/217) which will create the new test services for the case-api, so maybe you can target case-api-test now once that is merged?